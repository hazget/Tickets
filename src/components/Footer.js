import React from 'react';
import styled from 'styled-components';

const Footer = () => (
  <Container>
    Tickets Â© {new Date().getFullYear()}
  </Container>
);

export default Footer;

const Container = styled.footer`
  grid-area: footer;

  display: flex;
  justify-content: center;
  align-items: center;

  color: #4a4a4a;
  font-weight: 600;
`;