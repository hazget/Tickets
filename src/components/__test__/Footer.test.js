import React from 'react';
import Footer from '../Footer';
import renderer from 'react-test-renderer';

describe('Footer', () => {

  it("Footer - render correctly component", () => {
    const FooterComponent = renderer.create(<Footer/>).toJSON();
    expect(FooterComponent).toMatchSnapshot();
  });

});