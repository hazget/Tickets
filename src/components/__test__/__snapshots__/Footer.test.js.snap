// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer - render correctly component 1`] = `
<footer
  className="sc-bdVaJa eNrnpL"
>
  Tickets Â© 
  2022
</footer>
`;
